.eslintignore

```diff
@@ -1,2 +1,2 @@
 bin/
-lib/cli/blueprints/*/files/**
+lib/cli/blueprints/**/files/**
```

Comment:
This should stay a * since the 'files' directory is directly one level below in that folder, e.g. `blueprints/app/files`